scraper/src/youtube2zim/scraper.py

```diff
@@ -412,10 +407,6 @@ def run(self):
             logger.info("Finishing ZIM file…")
             self.zim_file.finish()
 
-        if not self.keep_build_dir:
```

Comment:
By default, `keep_build_dir` was False, so we want to keep the two lines below.

Reply:
You can amend commit 57f5a5d1fc1726e09db29d48b9ad8077d491fa29 directly

Reply:
Oops 😅 Fixed it in [1815f28](https://github.com/openzim/youtube/pull/262/commits/1815f280de9a13466b204f4d3015850a866747bf)